src/tests/build.proj

```diff
@@ -638,7 +644,9 @@
 
     <MSBuild
       Projects="Common\XUnitLogChecker\XUnitLogChecker.csproj"
-      Targets="Build" />
+      Targets="Publish"
+      Properties="_IsPublishing=true;Configuration=Release;PublishDir=$(XunitTestBinBase)/XUnitLogChecker;RuntimeIdentifier=$(RuntimeIdentifier)"
+      Condition="$(IsXUnitLogCheckerSupported)" />
```

Comment:
This is not on you obviously, but I wish we had standardized names for arguments like the RID. In tests.proj, it's `OutputRID`, and here it is `RuntimeIdentifier`.

Reply:
Yeah, I'm not sure what the reasoning is here. I would have to do some archeology to figure out our naming convention on this.